nest-forms-backend/src/ginis/ginis.controller.ts

```diff
@@ -91,22 +93,25 @@ export default class GinisController {
         `Form with id ${formId} does not have a ginisDocumentId`,
       )
     }
-    let document: DetailDokumentu | null = null
-    let owner: DetailReferenta | null = null
+    let wflDocument: SslDetailDokumentuWflDokument | null = null
+    let ownerDetail: GinDetailReferentaDetailReferenta | null = null
     let documentHistory: MappedDocumentHistory = []
     try {
-      document = await this.ginisAPIService.getDocumentDetail(ginisDocumentId)
-      owner = await this.ginisAPIService.getOwnerDetail(
-        document?.WflDokument[0]?.IdFunkceVlastnika,
+      const document =
+        await this.ginisAPIService.getDocumentDetail(ginisDocumentId)
+      wflDocument = document['Wfl-dokument']
+      const owner = await this.ginisAPIService.getOwnerDetail(
+        wflDocument['Id-funkce-vlastnika'],
       )
+      ownerDetail = owner['Detail-referenta']
       documentHistory = mapGinisHistory(document)
     } catch (error) {
       const errorToThrow =
-        error instanceof AxiosError && error.status === 404
+        error instanceof GinisError && error.axiosError?.status === 404
```

Comment:
Does `GinisError` extend `AxiosError`? If so, I would maybe call it `GinisAxiosError`, I wouldn't expect `axiosError` be present there.

Reply:
It doesn’t extend it, but if the underlying cause is `AxiosError`, it stores the error in the property, so you can access the original error